MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentReviewModel.swift

```diff
@@ -19,33 +19,21 @@ protocol PaymentReviewViewModelDelegate: AnyObject {
  View model class for review screen
   */
 public class PaymentReviewModel: NSObject {
-    var onDocumentUpdated: (() -> Void)?
 
-    var onExtractionFetched: (() -> Void)?
-    var onExtractionUpdated: (() -> Void)?
     var onPreviewImagesFetched: (() -> Void)?
     var reloadCollectionViewClosure: (() -> Void)?
     var updateLoadingStatus: (() -> Void)?
     var updateImagesLoadingStatus: (() -> Void)?
+
+    var onErrorHandling: (_ error: GiniMerchantError) -> Void = { _ in }
```

Comment:
Suggestion: lets use type ((_ error: GiniMerchantError) -> Void)? - to avoid sonar issues with empty blocks and increase perfomance